### PR TITLE
Add klicky modes

### PR DESCRIPTION
**Pull request by madisondegrezia**
- Addition of a parameter to the Klicky() function to allow a developer to run the click event listener in two different modes.
- Mode 1: Tracks all click events.
- Mode 2: Filters click events by the data-* attribute specified on a html tag.

### CODE REVIEW

File: src/klicky.ts [hunk 1]
```diff
@@ -1,3 +1,21 @@
-export const Klicky = () => {
+export const Klicky = (selector: string) => {
```
Comment (gschultz49) [inline src/klicky.ts:1]: Can you update this PR with the `data-*` selector work we talked about?

**Comment by madisondegrezia** [comment]
Hi Greg,
I didn't finish working on that, but I can take a quick look at it now. I
had the data-tracking property on the h1 tag, but am trying to figure out
how to compare that in the if statements and what would get passed in my
Klicky() function. Instead of the filter variable that I had passed in
Klicky(), should I pass data-* instead and not have a variable declared
above the function?

Best regards,
Madison DeGrezia

File: src/klicky.ts (hunk 1 above)
Comment (madisondegrezia) [reply src/klicky.ts]: PR has been updated.

**Review by gschultz49 — APPROVED** [review]
Nice job!

**Pull request merged**